.github/workflows/scheduled.yml

```
@@ -396,3 +397,80 @@ jobs:
           name: join-fuzzer-failure-artifacts
           path: |
             /tmp/join_fuzzer_repro
+
+
+  presto-java-aggregation-fuzzer-run:
+    runs-on: 8-core
+    container: ghcr.io/facebookincubator/velox-dev:presto-java
+    timeout-minutes: 120
+    # if: ${{ github.event_name != 'pull_request' }}
+    env:
+      CCACHE_DIR: "${{ github.workspace }}/.ccache/"
+      CCACHE_BASEDIR: "${{ github.workspace }}"
+      LINUX_DISTRO: "centos"
+    steps:
+
+      - name: "Restore ccache"
+        uses: assignUser/stash/restore@v1
+        with:
+          path: "${{ env.CCACHE_DIR }}"
+          key: ccache-presto-java-fuzzer
+
+      - name: "Checkout Repo"
+        uses: actions/checkout@v4
+        with:
+          path: velox
+          submodules: 'recursive'
+          ref: "${{ inputs.ref }}"
+
+      - name: Fix git permissions
+          # Usually actions/checkout does this but as we run in a container
+          # it doesn't work
+        run: git config --global --add safe.directory /__w/velox/velox/velox
+
+      - name: Zero Ccache Statistics
+        run: ccache -sz
+
+
+      - name: "Build"
+        run: |
+          cd velox
+          make debug NUM_THREADS="${{ inputs.numThreads || 8 }}" MAX_HIGH_MEM_JOBS="${{ inputs.maxHighMemJobs || 8 }}" MAX_LINK_JOBS="${{ inputs.maxLinkJobs || 4 }}" EXTRA_CMAKE_FLAGS="-DVELOX_ENABLE_ARROW=ON ${{ inputs.extraCMakeFlags }}"
```

```suggestion
        env:
          EXTRA_CMAKE_FLAGS: "-DVELOX_ENABLE_ARROW=ON ${{ inputs.extraCMakeFlags }}"
        run: |
          cd velox
          make debug NUM_THREADS="${{ inputs.numThreads || 8 }}" MAX_HIGH_MEM_JOBS="${{ inputs.maxHighMemJobs || 8 }}" MAX_LINK_JOBS="${{ inputs.maxLinkJobs || 4 }}" 
```
I noticed yesterday that passing extra flags like that doesn't work as the make file is only looking for the envvar.